optaweb-vehicle-routing-backend/src/test/java/org/optaweb/vehiclerouting/plugin/persistence/DistanceRepositoryIntegrationTest.java

```diff
@@ -1,75 +0,0 @@
-/*
```

Comment:
What is the reason for removing this test?

Reply:
Committed by mistake. Thanks for catching it!

Reply:
There is one more removed test - please see `LocationRepositoryIntegrationTest`